Date/Date.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include "Date.h"
+
+using namespace std;
+
+#define FEBRUARY 1
+#define MONTH_AMOUNT 12
+
+void Date::validate(int day, int month, int year) {
+    int daysInMonth[MONTH_AMOUNT] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
+    int indexMonth = month - 1;
+
+    if ( year % 400 == 0 || year % 4 == 0 && year % 100 != 0 ) {
+        daysInMonth[FEBRUARY] += 1;
+    }
+
+    if ( day > daysInMonth[indexMonth] || day < 1 ) {
```

Comment:
What if user passes month = 100500?
Oops...
You should test it better. On real job your team lead will be very sad if you will fix the same bug 1488 times